.gitignore

```diff
@@ -102,3 +102,6 @@ app/flavor*
 # Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
 !gradle-wrapper.jar
 
+/.favorites.json
+/.vscode/bookmarks.json
+.vscode/settings.json
```

Comment:
Is there something worth to commit into the repository overall, something not-my-computer related? You may generally add .vscode folder to gitignore. And if there is anything  important it can be anyway added with force add.